devlib/trace/perf.py

```diff
@@ -23,7 +24,8 @@
 from devlib.utils.misc import ensure_file_directory_exists as _f
 
 
-PERF_COMMAND_TEMPLATE = '{} stat {} {} sleep 1000 > {} 2>&1 '
+PERF_COMMAND_TEMPLATE = '{} {} {} {} sleep 1000 > {} 2>&1 '
```

Comment:
Basically, this is to save having to pull two files instead of one. Given that the output is generally parsed with a line parser looking for specific patterns, mixing in `stderr` lines usually don't impact things. The question is -- is there specific benefit to splitting `stdout` and `stderr` in this case?

>If we don't want that, isn't > {outfile} 2>&1 simply &> {outfile} or is there a subtle difference?

The `bash` man page specifically states they're semantically equivalent. I just end to write `2>&1` out if habit and because I keep forgetting that `&>` exists.

> Second question (tiny detail, out of curiosity), what is the final space in the string for?

Erm. Aesthetics? A subtle reference to the TBS [animated show](https://en.wikipedia.org/wiki/Final_Space)? \</joke> Seriously, I have no idea. It might be a hold-over form an earlier WA2 implementation where it might have mattered. But I don't see it serving a purpose in the current code. Well spotted.